test/e2e/go.mod

```diff
@@ -1,54 +1,56 @@
 module github.com/ovn-org/ovn-kubernetes/test/e2e
 
-go 1.20
+go 1.21
+
+toolchain go1.21.7
```

Comment:
why do we have toolchain coming here again? is your local version not aligned with what we need to 1.29.2?

Reply:
no my local go is on 1.21, im not sure why this is here, will remove it